models/user.xo.go

```diff
@@ -15,6 +15,7 @@ type User struct {
 	Auth         string         `json:"auth"`           // auth
 	NextExternal sql.NullString `json:"next_external"`  // next_external
 	ParentUserID sql.NullInt64  `json:"parent_user_id"` // parent_user_id
+	AwsCustomer  string 		`json:"aws_customer_identifier"` //AWS Customer identifier
```

Comment:
Did you modify the file by hand? The `.xo.go` files should only be modified by running `scripts/generate_model.sh`.